README.md

```diff
@@ -226,6 +226,22 @@ The hash for StremioServiceSetup.exe is <exe sha256 hash>
 
 If the `--quiet` flag is used together with `--dry-run` only the descriptor is printed to `STDOUT`. In case of error the error is printed to `STDERR` and `STDOUT` is blank.
 
+## Development
```

Comment:
I don't think this is needed, we are the ones updating the copyright headers, this doc suggest that contributors should do it
Also it could be done just by opening a code editor and do a project string replace